### ISSUE TITLE
Calling static functions passes `self` argument in compiled code

### ISSUE THREAD

**Issue by Trildar** [Perryvw/TypescriptToLua#39]
e.g.
```Typescript
declare namespace Foo {
  function bar(): void;
}

declare abstract class A {
  static x(): void;
}

Foo.bar();
A.x();
```
compiles to:
```Lua
Foo.bar(Foo)
A.x(A)
```

**Comment by Perryvw** [comment]
On static classes this is intended, as `self` becomes the class itself, you do this for example:
```typescript
class A {
    static myStaticField: string;
    static myFunction(): string {
        return this.myStaticField;
    }
}
```

For namespaces this does indeed appear to be a bug.

**Comment by Perryvw** [comment]
Fixed calls in 19d96cd5648307ed9553414514559527a9d6b554
